Automattic/wp-calypso
# Full Site Editing: hide taxonomies in templates

#### Changes proposed in this Pull Request

We don't want to expose this functionality because it's not useful
for the users. It's FSE internal implementation detail and changing
it could break front end rendering.

| Before | After |
| - | - |
| <img width="293" alt="Screenshot 2019-08-09 at 15 56 07" src="https://user-images.githubusercontent.com/1182160/62784246-5985bc00-babe-11e9-8057-755641557d0b.png"> | <img width="289" alt="Screenshot 2019-08-09 at 15 55 27" src="https://user-images.githubusercontent.com/1182160/62784256-61ddf700-babe-11e9-9437-f9663bd51901.png"> |

#### Testing instructions

1. Open header or footer for editing and verify that the Taxonomy accordion is no longer shown.
2. Make sure there are no regressions in editing and rendering.
